tests/Fixer/Import/FullyQualifiedStrictTypesFixerTest.php

```diff
@@ -0,0 +1,280 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\Fixer\Import;
+
+use PhpCsFixer\Tests\Test\AbstractFixerTestCase;
+
+/**
+ * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * @internal
+ *
+ * @covers \PhpCsFixer\Fixer\Import\FullyQualifiedStrictTypesFixer
+ */
+final class FullyQualifiedStrictTypesFixerTest extends AbstractFixerTestCase
+{
+    public function testImportedStrictTypesFixWithoutReturn()
```

Comment:
please use single test method with data provider.
info in names is nice indeed, yet you could as well put that info in data provider